.maestro/config.yaml

```diff
@@ -2,7 +2,7 @@ appStartup:
   enabled: false
 appSize:
   enabled: false
-disableRetries: true
+disableRetries: false
```

Comment:
You can remove the attribute, false is default value.

Reply:
https://github.com/duckduckgo/Android/pull/4528#discussion_r1595209963